source/_integrations/google_air_quality.markdown

```diff
@@ -0,0 +1,80 @@
+---
+title: Google Air Quality
+description: Instructions on how to integrate Google Air Quality into Home Assistant.
+ha_category:
+  - Sensor
+ha_release: 2025.7
+ha_iot_class: Cloud Poll
+ha_config_flow: true
+ha_codeowners:
+  - '@Thomas55555'
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+ha_domain: google_air_quality
+ha_quality_scale: bronze
+---
+
+The Google Air Quality integration provides Air quality data through the Google Air Quality API.
+
+## Prerequisites
+
+You need to configure developer credentials to allow Home Assistant to access your Google Account.
+These credentials are the same as the ones for [Google Sheets](/integrations/google_sheets), [Nest](/integrations/nest), [YouTube](/integrations/youtube), and [Google Mail](/integrations/google_mail).
+These are not the same as *Device Auth* credentials previously recommended for [Google Calendar](/integrations/google).
+Additionally you have to create a *Billing Account*. At the time of writing this documentation Google allows 10,000 API calls per month for free.
+
+{% warning %}
+This integration polls every 60 minutes. Here is an overview about how many API calls are performed per month in dependence of the amount of coordinates:
+
+Coordinates | API calls per month
+----------- | -------------------
+1           | 800
+2           | 1,600
+3           | 2,400
+4           | 3,200
+5           | 4,000
+10          | 8,000
+12          | 9,600
+
+Consider each restart of Home Assistant is an additional API call, per coordinate/entry.
+
+You can set up a [budget](https://cloud.google.com/billing/docs/how-to/budgets) for your billing account. This does not limit your costs, but you can get an alert, when reaching the budget.
+You may want to setup [disable billing with notifications](https://cloud.google.com/billing/docs/how-to/disable-billing-with-notifications). This can still produce costs in the between incurring costs and receiving budget notifications.
```

Comment:
```suggestion
You may want to setup [disable billing with notifications](https://cloud.google.com/billing/docs/how-to/disable-billing-with-notifications). This can still produce costs between incurring costs and receiving budget notifications.
```